experimental/web_dashboard/lib/src/auth/firebase.dart

```diff
@@ -2,9 +2,9 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:firebase_auth/firebase_auth.dart' hide FirebaseUser;
```

Comment:
i'm curious why `FirebaseUser` isn't triggering a lint warning

Reply:
Because it's not actually referenced in the file, you mean?